packages/embarkjs-ens/src/index.js

```diff
@@ -1,7 +1,12 @@
 /* global EmbarkJS Web3 namehash registerSubDomain require */
 
+import namehash from 'eth-ens-namehash';
+import ENSFunctions from './ENSFunctions';
+
 const __embarkENS = {};
 
+__embarkENS.ENSFunctions = ENSFunctions;
```

Comment:
Is this really useful? I see below that we just use the function with `ENSFunctions.registerSubDomain`

Reply:
we use this in embark-ens when importing this lib. before it used to be required both there and in this file. now it's only in this file